PoCs/kamaji/Metal3/delete-bmhs.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -eux
```

Comment:
Basically everything in this script is `|| true` so `-e` is meaningless. Might as well clean the `|| true` and remove `-e`